### PR TITLE
chore(features) enable CombinedServices by default

### LINKED ISSUE #3979: Remove `dbless-combined-services` from integration tests matrix
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem Statement

When `CombinedServices` becomes enabled by default, there's no need for a separate entry in the integration tests matrix. #3777 is to be released in 2.10.0 and according to our policy for its lifetime, it should be enabled by default in the next minor release, which most probably will be 2.11.0.

### Proposed Solution

Remove the `dbless-combined-services` entry from `.github/workflows/_integration_tests.yaml`.

### Additional information

_No response_

### Acceptance Criteria

- [ ] No redundant integration tests are run for `CombinedServices` feature gate

### PR DESCRIPTION
**Pull request by rainest**
**What this PR does / why we need it**:

Enables CombinedServices by default and updates test matrix to reflect this.

**Which issue this PR fixes**:

Fix #3979

**Special notes for your reviewer**:

See https://github.com/Kong/kubernetes-ingress-controller/issues/3979#issuecomment-1577388487. It looks like we meant to do this for 2.11, but the original issue only requests part of this change.

**PR Readiness Checklist**:

Complete these before marking the PR as `ready to review`:

- [x] the `CHANGELOG.md` release notes have been updated to reflect any significant (and particularly user-facing) changes introduced by this PR

### CODE REVIEW

File: FEATURE_GATES.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ Features that reach GA and over time become stable will be removed from this tab
 | CombinedRoutes   | `true`  | Beta  | 2.8.0   | TBD   |
 | GatewayAlpha     | `false` | Alpha | 2.6.0   | TBD   |
 | ExpressionRoutes | `false` | Alpha | 2.10.0  | TBD   |
-| CombinedServices | `false` | Alpha | 2.10.0  | TBD   |
+| CombinedServices | `true`  | Beta  | 2.10.0  | TBD   |
```
Comment (randmonkey) [inline FEATURE_GATES.md:64]: Should be `2.11.0` here?

**Review by randmonkey — COMMENTED** [review]
` feature-combinedservices=false` in `internal/manager/telemetry/telemetry_test.go` should be changed to ` feature-combinedservices=true`.

File: FEATURE_GATES.md (hunk 1 above)
Comment (czeslavo) [reply FEATURE_GATES.md:64]: This column is `Since` version, so it should stay as it is.

Comment (rainest) [reply FEATURE_GATES.md]: Yeah I wasn't sure about this, and it actually looks inconsistent, and some entries list initial addition whereas others list current phase.

Neither looks correct looking at the Kubernetes docs, which list each stage. I added 5e9c1b6b458e751ff03a039b4010ea97cbd56af6 to backfill those.

**Review by randmonkey — APPROVED** [review]

**Comment by pmalek** [comment]
How come the parser test - `TestParser_GoldenTests` - didn't fail on this PR? 🤔 👀 It's failing now on `main` (as it should without the necessary adjustment).

https://github.com/Kong/kubernetes-ingress-controller/actions/runs/5223254320

cc: @czeslavo

**Pull request merged**